regtests/t_spark_sql/src/spark_sql_views.sh

```diff
@@ -23,7 +23,7 @@ SPARK_BEARER_TOKEN="${REGTEST_ROOT_BEARER_TOKEN}"
 
 curl -i -X POST -H "Authorization: Bearer ${SPARK_BEARER_TOKEN}" -H 'Accept: application/json' -H 'Content-Type: application/json' \
   http://${POLARIS_HOST:-localhost}:8181/api/management/v1/catalogs \
-  -d '{"name": "spark_sql_views_catalog", "id": 100, "type": "INTERNAL", "readOnly": false, "properties": {"default-base-location": "file:///tmp/spark_sql_s3_catalog", "polaris.config.drop-with-purge.enabled": "true"}, "storageConfigInfo": {"storageType": "FILE", "allowedLocations": ["file:///tmp"]}}' > /dev/stderr
+  -d '{"name": "spark_sql_views_catalog", "id": 100, "type": "INTERNAL", "readOnly": false, "properties": [{"default-base-location": "file:///tmp/spark_sql_s3_catalog"}, {"polaris.config.drop-with-purge.enabled": "true"}], "storageConfigInfo": {"storageType": "FILE", "allowedLocations": ["file:///tmp"]}}' > /dev/stderr
```

Comment:
how about `"properties": {"default-base-location": "file:///tmp/spark_sql_s3_catalog", "polaris.config.drop-with-purge.enabled": "true"}`?

Reply:
Thank you for the catch @dimas-b ! This has been resolved.